extensions/amp-story-auto-analytics/0.1/test/test-amp-story-auto-analytics.js

```diff
@@ -53,5 +53,45 @@ describes.realWin(
       expect(autoAnalyticsEl.querySelector('amp-analytics[type="gtag"]')).to
         .exist;
     });
+
+    it('should add linker config', async () => {
+      await autoAnalyticsEl.whenBuilt();
+      const config = {
+        'linkers': {
+          'ampStoryAutoAnalyticsLinker': {
+            'ids': {
+              'cid': '${clientId}',
+            },
+            'enabled': true,
+            'proxyOnly': false,
+          },
+        },
+      };
+
+      const strConfig = JSON.stringify(config);
+      const configContents = strConfig.substr(1, strConfig.length - 2);
+
+      expect(autoAnalyticsEl.querySelector('script').textContent).to.contain(
+        configContents
+      );
+    });
+
+    it('should add cookieWriter config', async () => {
+      await autoAnalyticsEl.whenBuilt();
+      const config = {
+        'cookies': {
+          'ampStoryAutoAnalyticsCookies': {
+            'value': 'LINKER_PARAM(ampStoryAutoAnalyticsLinker, cid)',
+          },
+        },
+      };
+
+      const strConfig = JSON.stringify(config);
+      const configContents = strConfig.substr(1, strConfig.length - 2);
+
+      expect(autoAnalyticsEl.querySelector('script').textContent).to.contain(
+        configContents
+      );
+    });
```

Comment:
It's kind of a personal opinion but I'm not super convinced that configuration should be tested, unless you think someone might remove it inadvertently and you want an extra layer of "please don't do it it's super important". Totally up to you